pkg/zeek/escape.go

```diff
@@ -23,6 +25,30 @@ func Escape(data []byte) string {
 	return string(buf)
 }
 
+// EscapeUtf8 does the very same non-standard formatting of mixed-binary strings
+// that zeek does.  There is no way to disambiguate between random binary data
+// and a deliberate utf-8 string so it's left to the "presenation layer" to decide
+// how to format the data.  This would be remedied if a zeek string were a UTF-8
+// string and zeek had a bytes type to represent non-string data.
+func EscapeUtf8(data []byte) string {
+	var scratch [utf8.UTFMax]byte
+	var buf []byte
+	k := 0
+	for len(data) > 0 {
+		c := data[k]
```

Comment:
i think you can make this `c := data[0]` and drop the `k` declaration above?

Reply:
silly me